File: 1007 - diferenca/dif.pas
program difpas(Input,Output);

function diferenca( a:Integer; b:Integer; c:Integer; d:Integer  ):Integer;
begin
    diferenca := (a * b)-(c * d)
end;

var 
    a, b, c, d, diff  : Integer;

begin

    readLn(a);
    readLn(b);
    readLn(c);
    readLn(d);
    diff := diferenca(a, b, c, d);
    writeln ('DIFERENCA = ', diff);
end.

File: 1007 - diferenca/dif.rb
def diferenca( a, b, c, d)
    return ( (a * b) - (c * d))
end

a = gets.to_i
b = gets.to_i
c = gets.to_i
d = gets.to_i
diff = diferenca(a, b, c, d)
puts "DIFERENCA = %d" % [diff]